config/dev/local-images.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+  labels:
+    control-plane: controller-manager
+    app.kubernetes.io/name: deployment
+    app.kubernetes.io/instance: controller-manager
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: dpu-operator
+    app.kubernetes.io/part-of: dpu-operator
+    app.kubernetes.io/managed-by: kustomize
+spec:
+  template:
+    spec:
+      containers:
+      - command:
+        name: manager
+        env:
+        - name: DPU_DAEMON_IMAGE
+          value: localhost:5000/dpu-daemon:dev
+        - name: IMAGE_PULL_POLICIES
+          value: Always
+        image: localhost:5000/dpu-operator:dev
+        imagePullPolicy: Always
```

Comment:
@bn222 Why are we setting `imagePullPolicy: Always`? If the image is meant to be immutable (which it seems like it is based on the tag above) it might make more sense to do `imagePullPolicy: IfNotPresent`?

Reply:
when you build, you push to this image, then you deploy it with `make local-deploy`. If you overwrite the images, you want the latest. This patch is meant to make developer's life easy. That's why it also enables `imagePullPolicy: Always`